### ISSUE TITLE
Boost icon does not convey status well

### ISSUE THREAD

**Issue by neuhaus** [nolanlawson/pinafore#2215]
I am using the default settings, i.e. ozark theme.
I am unable to tell which toot has been boosted and which one hasn't. Please look at the two provided screenshots:
![551619B4-8272-4E7D-817F-AA3AA4F64CFD](https://user-images.githubusercontent.com/631764/202932893-71632c69-743b-4d95-a20e-d18e8d58f0eb.jpeg)

![7C9CBC4F-4764-4123-8E9D-B40457DD4C49](https://user-images.githubusercontent.com/631764/202932896-ee7cdded-2189-4de1-a07c-a11ac534e129.jpeg)

suggestion: use an outline for the boost icon on unboostet toots.

PS: The screenshot with the brighter boost icon is the unboosted one!

**Comment by nolanlawson** [comment]
Some of the themes have low contrast, it's true. Sorry about that. This has been a very common piece of feedback, so much that I added a new "contrast" tag to track issues with this.

**Comment by regulardesigner** [comment]
What about adding a small text "boosted" near the icon? It could convey the info without using a colour.

**Comment by NickColley** [comment]
Consolidating into https://github.com/nolanlawson/pinafore/issues/2230, thank you so much for raising.
